### PR TITLE
Fixed #40: Self Care Fragment Implemented with Grid View

### LINKED ISSUE #40: Add a Grid View for self-care section
-Add a grid view for self-care sections having 4 tiles.

### PR DESCRIPTION
**Pull request by MohitGupta121**
Fixed #40 

Self-Care Fragment Implemented with Grid View
Below is the screen recording of implementation:-

https://user-images.githubusercontent.com/76530270/158836218-15c86de0-be05-4cd2-a04b-a30abb6f581c.mp4

### CODE REVIEW

**Comment by MohitGupta121** [comment]
@aastha01chauhan can you please review this PR fro issue #40

**Comment by yellowHatpro** [comment]
Looks good, but there are a few things I would like to point out:
1. The icon for self-care doesn't reflect its purpose, please use some other relevant one.
2. The margin for the cards in the grid are not proper, please give a proper margin for the cards.
These changes should be sufficient for now, and we will be adding more later then.

**Comment by MohitGupta121** [comment]
> Looks good, but there are a few things I would like to point out:
> 
> 1. The icon for self-care doesn't reflect its purpose, please use some other relevant one.
> 2. The margin for the cards in the grid are not proper, please give a proper margin for the cards.
>    These changes should be sufficient for now, and we will be adding more later then.

1. Can you suggest some icon for that.
2. Which margin u r taking about not understood??

**Comment by MohitGupta121** [comment]
@yellowHatpro 
Can you suggest some icon for that.
Which margin u r taking about not understood??

**Comment by MohitGupta121** [comment]
@yellowHatpro @aastha01chauhan I can you please now review I changes that u requested

**Comment by MohitGupta121** [comment]
@yellowHatpro @aastha01chauhan  can u please add level tag in PR as per Gssoc

**Comment by MohitGupta121** [comment]
@yellowHatpro @aastha01chauhan is there any update on my PR can u please review

**Review by yellowHatpro — APPROVED** [review]
Good work.

File: app/src/main/res/layout/card_view_item_grid.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    app:cardCornerRadius="8dp"
```
[the remark this answers is not in the document]
Comment (MohitGupta121) [reply app/src/main/res/layout/card_view_item_grid.xml]: > please dont use dp for sizing, instead use intuit's sdp scaling.

Okay Sure, I used everywhere intuit's but there I missed I'll change that

[the remark this answers is not in the document]
Comment (MohitGupta121) [reply app/src/main/res/layout/card_view_item_grid.xml]: > please dont use dp for sizing, instead use intuit's sdp scaling.

oo it's already done by me it's last commit that's why.

**Pull request merged**